.github/workflows/deno.yml

```diff
@@ -27,10 +27,35 @@ jobs:
         run: |
           deno test --unstable --no-run denops/**/*.ts
 
+  typecheck-windows:
+    runs-on: windows-latest
+
+    steps:
+      - run: git config --global core.autocrlf false
+      - uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          path-type: inherit
+      - uses: actions/checkout@v2
+      - uses: denoland/setup-deno@main
+        with:
+          deno-version: ${{ env.DENO_VERSION }}
+      - name: Type check
+        run: |
+          deno test --unstable --no-run denops/**/*.ts
+        shell: msys2 {0}
+
   lint:
-    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - ubuntu-20.04
+    runs-on: ${{ matrix.os }}
 
     steps:
+      - if: startsWith(matrix.os, 'windows')
```

Comment:
*should*

Please use `runner.os` instead while that directly reflects what we need here.

https://docs.github.com/ja/actions/reference/context-and-expression-syntax-for-github-actions#job-context